Camera2D.gd

```diff
@@ -5,6 +5,9 @@ extends Camera2D
 
 
 
+
+
+
```

Comment:
Every PR has this extra space. I wonder why?

Reply:
I am an astronaut I like space